.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,11 +15,12 @@ Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-
 
 <!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->
 
-- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
 - Built on platform(s)
```

Comment:
We need to add that those are not mandatory.

Reply:
> Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers

Reply:
Assuming this is resolved since it already says that.